src/net.cpp

```diff
@@ -81,6 +81,9 @@ struct ListenSocket
 };
 }
 
+/** Services this node implementation cares about */
+static const uint64_t nRelevantServices = NODE_NETWORK;
```

Comment:
Going back to connecting to outbound nodes  maybe the best and easiest, solution is to only enforce NODE_NETWORK during IBD/IsInitialBlockdownload()...that's really the only time we care whether an outbound node has the full blockchain.  Once the chain is sync'd a  pruned node is just as useful to us as a NODE_NETWORK node.